docs/ref/views.txt

```diff
@@ -61,7 +61,13 @@ these with your own custom views, see :ref:`customizing-error-views`.
 The 404 (page not found) view
 -----------------------------
 
-.. function:: defaults.page_not_found(request, template_name='404.html')
+.. function:: defaults.page_not_found(request, exception, template_name='404.html')
+
+.. versionchanged:: 1.9
```

Comment:
I think these annotations should go at the bottom of the section (but tell me if you've seen other examples of them at the top like this)